### ISSUE TITLE
[BUG] make sure tracker works on GPU

### ISSUE THREAD

**Issue by davebulaval** [GRAAL-Research/poutyne#71]
Tracker fail because [missing](https://github.com/GRAAL-Research/poutyne/edit/dev/poutyne/framework/callbacks/tracker.py
) `.cpu` 
```
File "train.py", line 183, in <module>
    main()
  File "/home/fredy/anaconda3/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/fredy/anaconda3/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/fredy/anaconda3/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/fredy/anaconda3/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "train.py", line 176, in main
    callbacks=[tb_tracker])
  File "train.py", line 39, in train
    seed=seed)
  File "/home/fredy/git/poutyne/poutyne/framework/experiment.py", line 549, in train
    callbacks=callbacks)
  File "/home/fredy/git/poutyne/poutyne/framework/model.py", line 393, in fit_generator
    self._fit_generator_one_batch_per_step(epoch_iterator, callback_list)
  File "/home/fredy/git/poutyne/poutyne/framework/model.py", line 462, in _fit_generator_one_batch_per_step
    for step, (x, y) in train_step_iterator:
  File "/home/fredy/git/poutyne/poutyne/framework/iterators.py", line 84, in __iter__
    self.on_batch_end(step, batch_logs)
  File "/home/fredy/git/poutyne/poutyne/framework/callbacks/callbacks.py", line 234, in on_train_batch_end
    callback.on_train_batch_end(batch_number, logs)
  File "/home/fredy/git/poutyne/poutyne/framework/callbacks/tracker.py", line 134, in on_train_batch_end
    self.tracker.batch_statistic_upgrade(named_parameters)
  File "/home/fredy/git/poutyne/poutyne/framework/callbacks/tracker.py", line 44, in batch_statistic_upgrade
    batch_layer_abs_means.append(abs_value_layer_gradient.mean().numpy())
TypeError: can't convert CUDA tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.
```

**Comment by davebulaval** [comment]
To resolved:

- Add `.cpu`
- Add test with GPU

**Comment by freud14** [comment]
I think there are still tests missing for this issue.

**Comment by freud14** [comment]
Fixed by  #139.
